tests/atlocal.in

```diff
@@ -66,6 +66,13 @@ unset LANG
 # workaround to adjust the testsuite later:
 # FLAGS="-debug -Wall ${COBOL_FLAGS}"
 FLAGS="-debug -Wall ${COBOL_FLAGS} -fno-diagnostics-show-option"
+
+# MacOS workaround (HACK) to embed rpath during test
+if test "$GNUCOBOL_ENV_SETUP" != "1" -a "$GNUCOBOL_TEST_LOCAL" != "1"; then
+   # HACK, may need some version check, should likely be disabled if configure used --without-rpath
+   case "$OSTYPE" in darwin*) FLAGS="${FLAGS} -Q -Wl,-rpath,${abs_top_builddir}/libcob/.libs";; esac
```

Comment:
Does `abs_top_builddir` need quotes in case someone has spaces in the build path? But it's a workaround for the CI so I guess that may be ok.

Reply:
In theory, yes - as we now are back to "failing as before" I'll add it, just to check if this breaks the test runs again.

Reply:
Seems to work as good as before.